.github/workflows/codeql.yml

```diff
@@ -0,0 +1,42 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "feature/**", "hotfix/**", "develop", "main" ]
```

Comment:
I don't think this quite right. If someone has more experience, feel free to fix it. 

I haven't read this yet: https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/configuring-code-scanning#configuring-frequency

Reply:
Why wouldn't it be correct? We can always tweak this later if we notice that we need to expand it.

Reply:
I did not know before that this rpeo doesn't have a `main` branch so instead obviously we have to target `master`. Other than that we can probably avoid `feature` and `hotfix` on push and only scan when making PRs? Additionally, definitely suggest to add `release` branches.

Over all, I would suggest:
```
on:
  push:
    branches: [ "release/**", "develop", "master" ]
  pull_request:
    branches: [ "release/**", "develop", "master" ]
 ```

Reply:
So I guess we don't care about running this on feature branches? Should we then apply these patter to all repos running beehive?

Reply:
Reading the docs, I *think* this is supposed to be the target branch of the PR, not the source branch. So any PRs that will merge into develop, master, release. So feature branches, theoretically, would be included because they are going to merge into develop.

Reply:
Right I was thinking of long living ones where you have PRs targeting long living feature branches but we don't use that style of development that often and this will then run once you make the final PR and that is probably good enough.